### ISSUE TITLE
Rule proposal: console logging must have a string portion

### ISSUE THREAD

**Issue by bcowgill** [sindresorhus/eslint-plugin-unicorn#1248]
no-anonymous-console   every console.log, warn etc should have some identifying string as its first parameter.

I have been on projects where some rookie has put in console.log(a) while debugging something and then it was missed during peer review and some time later the unit tests just start spitting out some random variable value (or just undefined) with no indication of where the console message is coming from.  This can be very hard to track down and remove. 

## Fail

```js
console.log(o)
console.warn(o)
console.error(o)
etc...
```

## Pass

```js
console.log("value of", o)
console.warn(displayName, props)
```

**Comment by fisker** [comment]
When debugging, I normally use `console.log({foo})`, it's a fast way to indentify message.
